apps/website/screens/principles/data-visualization/DataVisualizationPage.tsx

```diff
@@ -0,0 +1,345 @@
+                    <Code>color-red-500</Code>
+                  </td>
+                  <td>#FE344F</td>
+                </tr>
+                <tr>
+                  <td>color-categorical-05</td>
+                  <td>
+                    <Code>color-yellow-800</Code>
+                  </td>
+                  <td>#947705</td>
+                </tr>
+                <tr>
+                  <td>color-categorical-06</td>
+                  <td>
+                    <Code>color-orange-700</Code>
+                  </td>
+                  <td>#C26C0A</td>
+                </tr>
+                <tr>
+                  <td>color-categorical-07</td>
+                  <td>
+                    <Code>color-purple-600</Code>
+                  </td>
+                  <td>#7D2FD0</td>
+                </tr>
+                <tr>
+                  <td>color-categorical-08</td>
+                  <td>
+                    <Code>color-blue-800</Code>
+                  </td>
+                  <td>#0067B3</td>
+                </tr>
+                <tr>
+                  <td>color-categorical-09</td>
+                  <td>
+                    <Code>color-green-900</Code>
+                  </td>
+                  <td>#135325</td>
+                </tr>
+                <tr>
+                  <td>color-categorical-10</td>
+                  <td>
+                    <Code>color-red-700</Code>
+                  </td>
+                  <td>#D0011B</td>
+                </tr>
+                <tr>
+                  <td>color-categorical-11</td>
+                  <td>
+                    <Code>color-yellow-900</Code>
+                  </td>
+                  <td>#624F04</td>
+                </tr>
+                <tr>
+                  <td>color-categorical-12</td>
+                  <td>
+                    <Code>color-orange-800</Code>
+                  </td>
+                  <td>#915108</td>
+                </tr>
+              </tbody>
+            </DxcTable>
+          </>
+        ),
+      },
+    ],
+  },
+  {
+    title: "How to apply our design tokens",
+    content: (
+      <>
+        <DxcParagraph>
+          Here is an example on how to apply our design tokens to a bar chart using the Cloudscape theming strategy.
+        </DxcParagraph>
+        <DxcParagraph>
+          For a more detailed explanation on how to style Cloudscape components, please refer to their{" "}
+          <Link href="https://cloudscape.design/foundation/visual-foundation/theming/" passHref legacyBehavior>
+            <DxcLink>theming documentation</DxcLink>
+          </Link>
+          . The complete list of data visualization color design tokens can be found{" "}
+          <Link href="https://cloudscape.design/foundation/visual-foundation/data-vis-colors/" passHref legacyBehavior>
+            <DxcLink>here</DxcLink>
+          </Link>
+          .
+        </DxcParagraph>
+        <Example example={themeBarChart} defaultIsVisible />
```

Comment:
![image](https://github.com/user-attachments/assets/269c09ef-5047-4462-987b-5643d52cd39d)

I am getting the following error in local caused by the example.

Reply:
I had this error the first time I added the example. You may need to delete node_modules and reinstall all the dependencies 😢

Reply:
You are right, it is no longer showing, sorry about that 😓